src/main/java/duke/TaskManager.java

```diff
@@ -0,0 +1,375 @@
+package duke;
+
+import exception.*;
```

Comment:
Instead of wildcard imports, declare your imports explicitly like in the [coding standard](https://se-education.org/guides/conventions/java/basic.html#package-and-import-statements).